packages/create-site/README.md

```diff
@@ -2,13 +2,9 @@
 
 `@jpmorganchase/mosaic-create-site` is a CLI script which scaffolds your very own Mosaic site from pre-defined generators.
 
-## Installation
-
-`yarn add -g @jpmorganchase/mosaic-create-site`
-
 ## Basic Commands
 
-### `yarn mosaic-create-site init`
+### `npx @jpmorganchase/mosaic-create-site init`
```

Comment:
I feel like `init` implies it is something that is required before you can create a site. I think having it here under basic commands might be a bit confusing for the reader. 

not necessarily in the scope of this PR